lib/features/email/presentation/email_controller.dart

```diff
@@ -73,8 +83,23 @@ class EmailController extends BaseController {
   void markAsEmailRead({required bool unread}) async {
     final accountId = mailboxDashBoardController.accountId.value;
     final presentationEmail = mailboxDashBoardController.selectedEmail.value;
-    if (accountId != null && presentationEmail != null) {
-      _markAsEmailReadInteractor.execute(accountId, presentationEmail.id, unread);
+    final mailboxCurrent = mailboxDashBoardController.selectedMailbox.value;
+    if (accountId != null && presentationEmail != null && mailboxCurrent != null) {
+      _markAsEmailReadInteractor.execute(accountId, presentationEmail.id, unread)
+          .then((result) => result
+            .map((success) {
+              if (success is MarkAsEmailReadSuccess) {
+                emailActionCurrent = MarkAsEmailReadAction(presentationEmail.id, mailboxCurrent);
+                _updateMailboxDashBoardActionForDesktop(emailActionCurrent);
+              }
+            })
+          );
+    }
+  }
+
+  void _updateMailboxDashBoardActionForDesktop(AppAction? appAction) {
```

Comment:
at the moment, please update with refresh for MailboxView and ThreadView, in the next sprint we will improve with `/changes`. Thanks